src/hotspot/share/cds/lambdaProxyClassDictionary.hpp

```diff
@@ -110,7 +110,7 @@ class DumpTimeLambdaProxyClassInfo {
   DumpTimeLambdaProxyClassInfo() : _proxy_klasses(NULL) {}
   void add_proxy_klass(InstanceKlass* proxy_klass) {
     if (_proxy_klasses == NULL) {
-      _proxy_klasses = new (ResourceObj::C_HEAP, mtClassShared)GrowableArray<InstanceKlass*>(5, mtClassShared);
+      _proxy_klasses = new (ResourceObj::C_HEAP, mtClassShared) GrowableArray<InstanceKlass*>(5, mtClassShared);
```

Comment:
Extra blank space added before “GrowableArray”.

Reply:
The usage style is like new (args...) type. This has been in hotspot and just want to keep with that.